test/int/cluster_test.go

```diff
@@ -175,6 +173,25 @@ func clusterPendingFunc(expectedState, expectedMessage string, reason workflow.C
 		}
 	}
 }
+func validateClusterUpdatingFunc() func(a mdbv1.AtlasCustomResource) {
+	isIdle := true
+	return func(a mdbv1.AtlasCustomResource) {
+		c := a.(*mdbv1.AtlasCluster)
+		// It's ok if the first invocations see IDLE
+		if c.Status.StateName != "IDLE" {
```

Comment:
as soon as the update is done the project may see "idle" state for some short time until the AO makes the API call for cluster update. 
This is not allowed in general though - only in the beginning, hence the "one side flag" (cannot be switched back)